packages/@sanity/vision/src/components/VisionGui.js

```diff
@@ -509,14 +498,20 @@ class VisionGui extends React.PureComponent {
           </Grid>
         </Header>
         <SplitpaneContainer flex={1}>
-          <SplitPane split="vertical" minSize={150} defaultSize={400}>
+          <SplitPane split="vertical" minSize={280} defaultSize={400} maxSize={-400}>
```

Comment:
I don’t understand how a negative number would work here. Maybe add a comment?

Reply:
I don't either, the syntax is kind of weird 🤡  I just tweaked it till it looked alright. Will add a comment though